dcos_installer/test_backend.py

```diff
@@ -67,6 +67,38 @@ def test_set_superuser_password(tmpdir):
         assert passlib.hash.sha512_crypt.verify('foo', config['superuser_password_hash'])
 
 
+def test_generate_node_upgrade_script(tmpdir):
```

Comment:
this makes sure the function is not crashing and burning, ill add check for return 0 . 

I would like suggestions as to what makes sense to add as unit tests.